pkgs/development/libraries/libnest2d/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, cmake, clipper, nlopt, boost, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "4.8.0";
+  pname = "libnest2d";
+  format = "other";
```

Comment:
This doesn't belong here, right?

Reply:
Sorry, I'm not sure what you mean. Should this file be in another folder?

Reply:
`format = "other"` is only for `buildPythonPackage` as far as I know.

Reply:
```suggestion
```

Please just remove the `format = "other"` line.